PSF-LICENSE

```diff
@@ -0,0 +1,254 @@
+A. HISTORY OF THE SOFTWARE
```

Comment:
Maybe this license file can be placed in the Lib folder, together with a readme indicating what it applies to.